config/version_git.h.in

```diff
@@ -0,0 +1,9 @@
+#define NVIM_VERSION_MAJOR @NVIM_VERSION_MAJOR@
```

Comment:
I think the reason was that no "ordinary" source file is suppored to include `auto/config.h` directly, rather just at a specific place in `vim.h`. But of course it doesn't hurt to be consistent so I won't mind adding them.